src/main/java/net/openhft/chronicle/wire/CharConverter.java

```diff
@@ -17,6 +17,7 @@
  */
 package net.openhft.chronicle.wire;
 
+@Deprecated(/* for removal in x.27, use LongConverter */)
```

Comment:
I would advise against bundling this deprecation in here as other devs might miss it. Potentially better in a separate review.

Reply:
In future, perhaps, in this case, it explains why there is no Javadoc because it hasn't been used for a long time, probably since 2015.